util/helm/helm_test.go

```diff
@@ -157,14 +182,6 @@ func TestHelmArgCleaner(t *testing.T) {
 	}
 }
 
-func TestVersion(t *testing.T) {
```

Comment:
Hm, why has this test been zapped?

Reply:
Restored the test.

The `helm version` command output has changed and helm2 regex no longer able to extract semantic version. For Kustomize we had the same problem and decided to just return version command output instead of supporting every version. I've made the same change in `helm.Version` method and updated the test accordingly.